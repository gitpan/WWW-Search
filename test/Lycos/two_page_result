http://www.lycos.com/cgi-bin/pursuit?adv=1&matchmode=or&maxhits=100&query=LSAM+ISI
http://www.lycos.com/cgi-bin/pursuit?cat=lycos&first=101&maxhits=100&bq=2&br=2&query=LSAM+ISI&matchmode=or&adv=1
http://www.lycos.com/cgi-bin/pursuit?cat=lycos&first=201&maxhits=100&bq=2&br=2&query=LSAM+ISI&matchmode=or&adv=1
http://www.lycos.com/cgi-bin/pursuit?cat=lycos&first=301&maxhits=100&bq=1&br=1&query=LSAM+ISI&matchmode=or&adv=1
http://www.lycos.com/cgi-bin/pursuit?cat=lycos&first=401&maxhits=100&bq=1&br=1&query=LSAM+ISI&matchmode=or&adv=1
http://www.lycos.com/cgi-bin/pursuit?cat=lycos&first=501&maxhits=100&bq=1&br=1&query=LSAM+ISI&matchmode=or&adv=1
http://www.lycos.com/cgi-bin/pursuit?cat=lycos&first=601&maxhits=100&bq=1&br=1&query=LSAM+ISI&matchmode=or&adv=1
http://www.lycos.com/cgi-bin/pursuit?cat=lycos&first=701&maxhits=100&bq=1&br=1&query=LSAM+ISI&matchmode=or&adv=1
http://www.lycos.com/cgi-bin/pursuit?cat=lycos&first=801&maxhits=100&bq=1&br=1&query=LSAM+ISI&matchmode=or&adv=1
