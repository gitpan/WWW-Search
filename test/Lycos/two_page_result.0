<HTML>
<HEAD>
<TITLE>Lycos Search: LSAM ISI</TITLE>
</HEAD>
<!--begin resultheader.inc-->

<BODY BGCOLOR="#FFFFFF" link="#000088">

<TABLE BORDER="0" CELLSPACING=0 CELLPADDING=0 WIDTH=602>
<TR>
<TD WIDTH=125 BGCOLOR="#000000" VALIGN=TOP ROWSPAN=9>


<a href="http://www.lycos.com/"><IMG SRC="/assist/graphics/new-logo.gif" width=125 height=48 alt="Lycos" border=0></a>
<a name="refine"><IMG SRC="/assist/graphics/res-icon.gif" height=76 width=125 alt="Search Results"></A><BR>


<!-- GUIDEBAR -->	
<br>
<TABLE WIDTH=125 BORDER=0 CELLSPACING=4 CELLPADDING=0 VALIGN=TOP BGCOLOR="#000000">
<TR VALIGN=TOP>	<TD WIDTH=10 ROWSPAN=6>&#160;</TD>
		<TD VALIGN=TOP WIDTH=115 BGCOLOR="#FFCC33">&#160;<A HREF="http://www.lycos.com"><FONT FACE="ARIAL,HELVETICA,MS SANS SERIF,SANS-SERIF" COLOR="#000000" SIZE="-1"><B>Lycos Home</B></FONT></A></TD></TR>
<TR>		<TD VALIGN=TOP WIDTH=115 BGCOLOR="#FFCC33">&#160;<A HREF="http://www.lycos.com/webguides/webguides.html"><FONT FACE="ARIAL,HELVETICA,MS SANS SERIF,SANS-SERIF" COLOR="#000000" SIZE="-1"><B>Web Guides</B></FONT></A></TD></TR>
<TR>		<TD VALIGN=TOP WIDTH=118 bgcolor="#FFCC33" valign="top">
<FONT FACE="ARIAL,HELVETICA,MS SANS SERIF,SANS-SERIF" COLOR="#000000" SIZE="-1">
&nbsp;<B><A HREF="/autos/"><B>Autos</B></A>
<BR>&nbsp;<B><A HREF=/business/><B>Business</B></A>
<BR>&nbsp;<B><A HREF=/careers/><B>Careers</B></A>
<BR>&nbsp;<B><A HREF=/computers/><B>Computers</B></A>
<BR>&nbsp;<B><A HREF=/education/><B>Education</B></A>
<BR>&nbsp;<B><A HREF=/entertainment/><B>Entertainment</B></A>
<BR>&nbsp;<B><A HREF=/fashion/><B>Fashion</B></A>
<BR>&nbsp;<B><A HREF=/government/><B>Government</B></A>
<BR>&nbsp;<B><A HREF=/health/><B>Health</B></A>
<BR>&nbsp;<B><A HREF=/kids/><B>Kids</B></A>
<BR>&nbsp;<B><A HREF=/money/><B>Money</B></A>
<BR>&nbsp;<B><A HREF=/news/><B>News</B></A>
<BR>&nbsp;<B><A HREF=/people/><B>People</B></A>
<BR>&nbsp;<B><A HREF=/shopping/><B>Shopping</B></A>
<BR>&nbsp;<B><A HREF=/space/><B>Space/Sci-Fi</B></A>
<BR>&nbsp;<B><A HREF=/sports/><B>Sports</B></A>
<BR>&nbsp;<B><A HREF=/travel/><B>Travel</B></A>
</font>
</td>
</tr>
<TR>		<TD VALIGN=TOP WIDTH=115 BGCOLOR="#FFCC33">&#160;<A HREF="http://lycospro.lycos.com"><FONT FACE="ARIAL,HELVETICA,MS SANS SERIF,SANS-SERIF" COLOR="#000000" SIZE="-1"><B>Search</B></FONT></A></TD></TR>
<TR>		<TD VALIGN=TOP WIDTH=115 BGCOLOR="#FFCC33">&#160;<A HREF="http://www.lycos.com/help/"><FONT FACE="ARIAL,HELVETICA,MS SANS SERIF,SANS-SERIF" COLOR="#000000" SIZE="-1"><B>Help</B></FONT></A></TD></TR>
<TR>		<TD VALIGN=TOP WIDTH=115 BGCOLOR="#FFCC33" VALIGN=TOP>&#160;<A HREF="http://echomail.lycos.com/echomail"><FONT FACE="ARIAL,HELVETICA,MS SANS SERIF,SANS-SERIF" COLOR="#000000" SIZE="-1"><B>Feedback</B></FONT></A></TD>

<TD WIDTH=2 BGCOLOR="#000000" VALIGN=TOP ROWSPAN=5>&#160;</TD>
	</TR>
	</TABLE>
	</TD>

<!---- END GUIDEBAR---->


<TD WIDTH=476 VALIGN=TOP>

<!--begin flag.inc-->

<!-- SERVICEBAR-->

<TABLE BORDER=0 CELLSPACING=0 CELLPADDING=0 width=470>
<TR valign=bottom HEIGHT=28>
<td bgcolor="#6699FF" WIDTH=477 valign=bottom  HEIGHT=28>
<MAP NAME=service>
<AREA SHAPE=rect COORDS="6,0, 54,14" HREF="http://point.lycos.com/categories/">
<AREA SHAPE=rect COORDS="57,0, 115,14" HREF="http://cityguide.lycos.com/">
<AREA SHAPE=rect COORDS="119,0, 181,14" HREF="http://www.lycos.com/pplfndr.html">
<AREA SHAPE=rect COORDS="184,0, 243,14" HREF="http://www.stockfind.newsalert.com">
<AREA SHAPE=rect COORDS="249,0, 307,14" HREF="http://www.lycos.com/roadmap.html">
<AREA SHAPE=rect COORDS="311,0, 386,14" HREF="http://www.lycos.com/cgi-bin/nph-bounce?gtesearch|http://yp.gte.net/lsearch">
<AREA SHAPE=rect COORDS="391,0, 467,14" HREF="http://www.lycos.com/ups/bridge.html">
<AREA SHAPE=default  HREF="http://www.lycos.com/">
</MAP>

<a href="/assist/maps/service.map"><IMG SRC="/assist/graphics/servicebar.gif" HEIGHT=18 WIDTH=473 alt="Lycos Services" BORDER=0 usemap="#service" ISMAP></a>
</TD>
</TR>
<TR><TD>&nbsp;</TD></TR>

<!-- END SERVICEBAR-->

<TR>
<!--end flag.inc-->

<!--begin imap.inc-->

<!-- AD -->
<TD ALIGN=LEFT>
<FONT SIZE=-2 FACE="helvetica, arial, ms sans serif, sans-serif">
<center><p><a href="/cgi-bin/nph-adclick.exe/CID=0000389c0000000300000000/ST=lycos/SV=search/A=.lycos.search_lycos."><img  src="http://ads.lycos.com/ads/lycos-house14.gif" alt="Lycos StockFind...Or Else."  width=468 height=60></a><br><a href="/cgi-bin/nph-adclick.exe/CID=0000389c0000000300000000/ST=lycos/SV=search/A=.lycos.search_lycos.">Lycos StockFind...Or Else.</a></center>
</FONT>
</TD>
</TR>
</TABLE>
<!-- END AD-->



<!-- CONTENT-->
<P>


<!-- HTML Search Form starts here -->

<TABLE BGCOLOR="#6699FF" CELLSPACING=0 CELLPADDING=1 BORDER="0">
<TR>
<TD ALIGN=CENTER VALIGN=MIDDLE WIDTH=473 COLSPAN=2>


	<FORM ACTION="/cgi-bin/pursuit" METHOD=GET>
	<INPUT TYPE="hidden" NAME="adv" VALUE="0">
	<FONT SIZE=-1  FACE="HELVETICA, ARIAL, SANS SERIF, MS SANS SERIF"><B>You Searched</B>&nbsp;<SELECT NAME="cat">
	<OPTION SELECTED value="lycos">The Web
	<OPTION value="sounds">Sounds
	<OPTION value="graphics">Pictures
	<OPTION value="point">TOP 5%
	<OPTION value="geo">Personal Homepages
	<OPTION value="bn">Books
	<OPTION VALUE="stockfind">Stock Symbol</SELECT>
	&nbsp;for&nbsp;<SELECT NAME="matchmode">
	<OPTION SELECTED value="and">All the words
	<OPTION value="or">Any of the words
	
	</SELECT>
	</FONT>
</TD>
</TR>

<TR>
<TD ALIGN=CENTER VALIGN=MIDDLE WIDTH=472 COLSPAN=2>

    <INPUT TYPE="text" NAME="query" VALUE="LSAM ISI" SIZE=50>	
</TD>
</TR>

<TR ALIGN=CENTER VALIGN=TOP>
<TD ALIGN=CENTER VALIGN=BOTTOM>       
	<A HREF="http://lycospro.lycos.com/lycospro-nojava.html"><IMG SRC="/assist/graphics/searchresults/lycosprolink.gif" WIDTH=261 HEIGHT=23 BORDER="0"></A>
</TD>
<TD ALIGN=CENTER VALIGN=TOP>
	<INPUT TYPE="image" SRC="/assist/graphics/gogetit(R).gif" height=22 width=72 BORDER=0>
</TD>
</TR>
</FORM>		
</TABLE>


<!-- End HTML Search Form -->

<!-- "GET MORE ON YOUR SEARCH" TABLE -->

<TABLE WIDTH=473 BORDER=0 CELLSPACING=1 CELLPADDING=2>
<TR>
<TD BGCOLOR="#6699FF" COLSPAN=3>
<B><FONT FACE="Helvetica, Arial, MS Sans Serif, sans-serif" COLOR="000000" SIZE=-1>Get more on: LSAM ISI</font>&nbsp;&nbsp;&nbsp;<FONT FACE="Helvetica, Arial, MS Sans Serif, sans-serif" COLOR="000000" SIZE=-2>Choose from the categories below.</FONT></B>
</TD>
</TR>
<TR>
<TD BGCOLOR="#99CCFF" VALIGN=MIDDLE> <a href="/cgi-bin/pursuit?cat=bn&amp;query=LSAM%20ISI"> <FONT FACE="helvetica, arial, ms sans serif, sans-serif" COLOR="#000000" SIZE=-1><B>Search Books</B></FONT></A>
</TD>
<TD BGCOLOR="#99CCFF" VALIGN=MIDDLE> <a href="/cgi-bin/pursuit?cat=graphics&amp;query=LSAM%20ISI"> <FONT FACE="helvetica, arial, ms sans serif, sans-serif" COLOR="#000000" SIZE=-1><B>Search Pictures</B></FONT></A>
</TD>
<TD BGCOLOR="#99CCFF" VALIGN=MIDDLE> <a href="/cgi-bin/pursuit?cat=geo&amp;query=LSAM%20ISI"> <FONT FACE="helvetica, arial, ms sans serif, sans-serif"A COLOR="#000000" SIZE=-1><B>Search Personal Homepages</B></FONT></A>
</TD>
</TR>
<TR>
<TD BGCOLOR="#99CCFF" VALIGN=MIDDLE> <a href="/cgi-bin/pursuit?cat=point&amp;query=LSAM%20ISI"> <FONT FACE="helvetica, arial, ms sans serif, sans-serif" COLOR="#000000" SIZE=-1><B>Search TOP 5% </B></FONT></A>
</TD>
<TD BGCOLOR="#99CCFF" VALIGN=MIDDLE> <a href="/cgi-bin/pursuit?cat=sounds&amp;query=LSAM%20ISI"> <FONT FACE="helvetica, arial, ms sans serif, sans-serif" COLOR="#000000" SIZE=-1><B>Search Sounds</B></FONT></A>
</TD>
<TD BGCOLOR="#FFCC00" VALIGN=MIDDLE> <a href="http://www.lycos.com/cgi-bin/nph-bounce?gtesearch|http://yp.gte.net/lsearch"> <FONT FACE="helvetica, arial, ms sans serif, sans-serif" COLOR="#000000" SIZE=-1><B>Search Yellow Pages</B></FONT></A>
</TD>
</TR>
</table>

</td>
</TR>
<!-- START CONTENT-->
<TR>
<TD VALIGN=TOP>
&#160;&#160;<FONT SIZE=2 FACE=HELVETICA,ARIAL>Browse thousands of <A HREF="/resources/investors/"><B>cool tools for Investors!</B></A></FONT>

<br>
&nbsp;<FONT SIZE=-1 FACE="helvetica, arial, ms sans serif, sans-serif"><b> 1-10 of 11 relevant results</b>
 
 
 


<!-- Formatted results start here -->

<BR>
&nbsp;<FONT FACE="Helvetica, Arial, MS Sans Serif, sans-serif" size="-1">
<A HREF="/cgi-bin/pursuit?first=1&cat=lycos&matchmode=and&results=terse&adv=0&query=LSAM%20ISI">Just the links</A>&nbsp;&nbsp;<A HREF="/cgi-bin/pursuit?first=1&cat=lycos&matchmode=and&results=default&adv=0&query=LSAM%20ISI">Standard Descriptions</A>&nbsp;&nbsp;<A HREF="/cgi-bin/pursuit?first=1&cat=lycos&matchmode=and&results=verbose&adv=0&query=LSAM%20ISI">Detailed Descriptions</A></font>	
</TD></TR>

<TR><TD>
<TABLE CELLPADDING=4 CELLSPACING=0 BORDER=0>
<TR><TD>
<P>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>1) <a href="http://www.isi.edu/div7/cities/people.html">LSAM People</a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>[LSAM logo] LSAM People Joe Touch. Project Leader. Coordinating intelligent bandwidth and mobility tasks, performing research on Intelligent Bandwidth.</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://www.isi.edu/div7/cities/people.html [100%]</font><p>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>2) <a href="http://www.isi.edu/lsam/people.html">LSAM People</a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>[LSAM logo] LSAM People Joe Touch. Project Leader. Coordinating intelligent bandwidth and mobility tasks, performing research on Intelligent Bandwidth.</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://www.isi.edu/lsam/people.html [100%]</font><p>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>3) <a href="http://gin.isi.edu:1080/cgi-bin/gra/application">GRA Application</a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>USC/ISI GRA Application ----------- Thanks for your interest in the research projects in division 7. To apply for a position with one of the projects,.</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://gin.isi.edu:1080/cgi-bin/gra/application [86%]</font><p>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>4) <a href="http://www.isi.edu/div7/cities/construction.html">IB Under Constuction</a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>Under Construction ----------- This page is currently under construction. Watch this space for updated information in the near future. Information on .</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://www.isi.edu/div7/cities/construction.html [64%]</font><p>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>5) <a href="http://www.isi.edu/lsam/construction.html">IB Under Constuction</a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>Under Construction ----------- This page is currently under construction. Watch this space for updated information in the near future. Information on .</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://www.isi.edu/lsam/construction.html [64%]</font><p>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>6) <a href="http://www.isi.edu/ib/plan.html">IB Research Plan</a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>LOGO IB Research Plan ----------- For the experiments in this research, we will make use of traffic generators that emulate WWW browser request stream.</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://www.isi.edu/ib/plan.html [64%]</font><p>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>7) <a href="http://www.isi.edu/ib/discussion.html">IB Discussion</a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>LOGO IB Discussion ----------- Response time and bandwidth are key resources in this project. Current Internet user interfaces, such as Web browsers, .</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://www.isi.edu/ib/discussion.html [64%]</font><p>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>8) <a href="http://gost.isi.edu/brian/resume/work.html">Brian Tung </a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>Brian Tung Computer Scientist at ISI I started working at ISI in November 1994. For the moment, I'm splitting my time between two projects: Security I.</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://gost.isi.edu/brian/resume/work.html [64%]</font><p>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>9) <a href="http://www.isi.edu/~katia/">Katia Obraczka, USC Information Science Institute </a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>Katia Obraczka Address USC Information Science Institute 4676 Admiralty Way Marina del Rey, CA 90212 voice: (310) 822-1511 fax: (310) 823-6714 e-mail:.</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://www.isi.edu/~katia/ [64%]</font><p>
<FONT SIZE=-1 FACE=ARIAL, HELVETICA><b>10) <a href="http://www.isi.edu/~touch/projects.html">Joe's Projects </a></b></font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>Joe Touch - Projects ----------- ATOMIC-2 Gigabit Networking Bringing gigabit LAN performance into the workstation. My research addresses fault-tolera.</font>
<br><FONT SIZE=-1 FACE=ARIAL, HELVETICA>http://www.isi.edu/~touch/projects.html [64%]</font><p>

<p>


<!-- end formatted results -->
</TD></TR></TABLE>

<!--END TABLE DATA AND TABLE ROW GO HERE-->



<tr align=top width=476>
<td valign=center>
<TABLE CELLPADDING=4 CELLSPACING=0 BORDER=0>
<TR>
<TD>
<!-- Previous/Next page/Refine Search stufff... -->

<P><BR>

<FONT SIZE=-1 FACE="helvetica, arial, ms sans serif, sans-serif">Previous Page &nbsp;&nbsp; <A HREF="/cgi-bin/pursuit?first=11&cat=lycos&matchmode=and&adv=0&query=LSAM%20ISI">Next Page</A> &nbsp;&nbsp; <A HREF="#refine">Refine Your Search</A></font>


<p>
<FONT SIZE="-1" FACE="helvetica, arial, ms sans serif, sans-serif">Go to page:&nbsp;&nbsp;<B><FONT SIZE=+1>1</FONT>
&nbsp;.&nbsp;<A HREF="/cgi-bin/pursuit?first=11&cat=lycos&matchmode=and&adv=0&query=LSAM%20ISI">2</A>
</B> 
<p>
 


</TD>
</TR>






<TR>
<TD ALIGN=LEFT>

<FONT SIZE=-2>
<center><a href="/cgi-bin/nph-adclick.exe/CID=0000389c0000000300000000/ST=lycos/SV=search/A=.lycos.search_lycos."><img  src="http://ads.lycos.com/ads/lycos-house14.gif" alt="Lycos StockFind...Or Else."  width=468 height=60></a><br><a href="/cgi-bin/nph-adclick.exe/CID=0000389c0000000300000000/ST=lycos/SV=search/A=.lycos.search_lycos.">Lycos StockFind...Or Else.</a></center>
</FONT>

</TD>
</TR>
</BODY>


<!-- "New Search" link changed to "Lycos Home": -->

<TR>
<TD>

<FONT SIZE="-2" FACE="helvetica, arial, ms sans serif, sans-serif">
<nobr><a href="http://www.lycos.com/">Lycos Home</a> <strong><font size=+1>&#46;</font></strong> 
<a href="http://topnews.lycos.com/">TopNews</a> <strong><font size=+1>&#46;</font></strong> 
<a href="http://point.lycos.com/categories/">TOP 5% </a> <strong><font size=+1>&#46;</font></strong>
<a href="http://cityguide.lycos.com/">City Guide</a> <strong><font size=+1>&#46;</font></strong>
<a href="http://www.stockfind.newsalert.com/">StockFind</a></nobr><BR>
<nobr><a href="http://www.lycos.com/pplfndr.html">PeopleFind</a> <strong><font size=+1>&#46;</font></strong>
<a href="http://www.companiesonline.com/">Companies Online</a> <strong><font size=+1>&#46;</font></strong>
<a href="http://www.lycos.com/roadmap.html">Road Maps</a> <strong><font size=+1>&#46;</font></strong>
<a href="http://www.lycos.com/software/">Software</a> <strong><font size=+1>&#46;</font></strong>
<a href="http://www.lycos.com/info/">About Lycos</a> <strong><font size=+1>&#46;</font></strong>
<a href="http://www.lycos.com/help/">Help</a></nobr><BR>
<a href="http://www.lycos.com/cgi-bin/nph-bounce?gtesearch|http://yp.gte.net/lsearch">GTE Yellow Pages</a> <strong><font size=+1>&#46;</font></strong>
<a href="http://www.lycos.com/addasite.html">Add Your Site to Lycos</a> <strong><font size=+1>&#46;</font></strong> 
<a href="http://www.lycos.com/lycosinc/advertising.html">Advertise with Lycos</a> <strong><font size=+2>&#46;</font></strong>
<a href="http://www.lycos.com/lycosinc/lycosjobs.html">Jobs4You</a> <strong><font size=+1>&#46;</font></strong>
<a href="http://www.lycos.com/new2net.html">New2Net</a></font></nobr>
</FONT>

<FONT SIZE="-2" FACE="helvetica, arial, ms sans serif, sans-serif">
<P><nobr><a href="http://www.lycos.de">Lycos Germany</a> <strong><font size=+2>&#46;</font></strong> 
<a href="http://www.lycos.se">Lycos Sweden</a> <strong><font size=+2>&#46;</font></strong> <a href="http://www-uk.lycos.com"> Lycos UK</a></nobr>
<P> <font size="-2">
<A HREF="http://www.lycos.com/lycosinc/legal.html">Copyright</A>&copy; 1997 Lycos,Inc.<BR>
All Rights Reserved.<BR>
Lycos<font size="-3"><sup>&#174;</sup></font> is a registered trademark of<BR>
Carnegie Mellon University<BR>
<A HREF="http://www.lycos.com/contact.html">Questions &amp; Comments</A><BR>
<A HREF="http://www.lycos.com/lycosinc/legal.html">Terms and conditions</A>
</font>

<!-- END FOOTER-->
</FONT>
</td>
</tr>
</table>


</TD></TR></TABLE>


</HTML>
