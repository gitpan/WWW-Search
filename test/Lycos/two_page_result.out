http://www.isi.edu/lsam/people.html
http://www.isi.edu/~wls/
http://www.isi.edu/lsam/
http://www.isi.edu/lsam/tools/WWW_SEARCH/
http://www.isi.edu/div7/gra/
http://www.isi.edu/~larse/
http://www.isi.edu/~surya/
http://www.isi.edu/div7/research-top.html
http://www.isi.edu/~katia/http-perf-short.html
http://www.isi.edu/~touch/projects.html
http://www.isi.edu/ib/
http://www.isi.edu/ib/plan.html
http://www.isi.edu/~johnh/PAPERS/Visweswaraiah97b.html
http://www.isi.edu/~johnh/SOFTWARE/
http://www.isi.edu/~katia/
http://www.isi.edu/ib/discussion.html
http://www.isi.edu/~johnh/SOFTWARE/APACHE/apache_1.0_isi_mmap_960524.patch
http://www.isi.edu/lsam/construction.html
http://www.isi.edu/~ahughes/
http://www.mbone.com/lists/rem-conf.1997Q1/0109.html
